src/renderer/components/FormComponents/FormSelectDetailed/index.tsx

```diff
@@ -25,7 +25,7 @@ const FormSelectDetailed: FC<ComponentProps> = ({hideError = false, label, requi
         onChange={handleChange}
         value={selectedOption}
       />
-      {hideError ? null : renderFormError(name, className)}
+      {hideError ? renderFormEmptyError() : renderFormError(name, className)}
```

Comment:
Created this function to give `hideError= true` and `hideError= false AND no actual error` the same height. Otherwise when we had `hideError= true` and it returned `null` it was didn't render the error div at all and was squishing together.

### renderFormEmptyError()
<img width="409" alt="Screen Shot 2020-08-15 at 5 39 22 PM" src="https://user-images.githubusercontent.com/8547538/90324049-afd1da00-df1e-11ea-9053-577e08617a12.png">

### null
<img width="397" alt="Screen Shot 2020-08-15 at 5 39 46 PM" src="https://user-images.githubusercontent.com/8547538/90324052-c0825000-df1e-11ea-9f0b-6911a1069a44.png">